packages/clerk-js/src/core/clerk.ts

```diff
@@ -181,6 +186,24 @@ export class Clerk implements ClerkInterface {
   #pageLifecycle: ReturnType<typeof createPageLifecycle> | null = null;
   #touchThrottledUntil = 0;
 
+  public __internal__createPublicCredentials:
```

Comment:
Also, `__internal_XXX` needs a single underscore before the variable name.
```suggestion
  public __internal_createPublicCredentials:
```